src/data/markdown/translated-guides/en/07 Misc/09 Glossary.md

```diff
@@ -1,189 +1,203 @@
 
-### Horizontal Scalability
+<dt id="endurance-testing">Endurance testing</dt>
+<dd>A synonym for [soak testing](#soak-test).</dd>
 
-**Horizontal Scalability** is a trait describing to what degree a system under test’s performance and capacity may be increased by adding more nodes (servers or computers for instance).
+<dt id="executor">Executor</dt>
+<dd>An property of a [scenario](#scenario) that configures VU behavior. You can use executors to configure whether to designate iterations as shared between VUs or to run per VU, or to configure or whether the VU concurrency is constant or changing</dd>
 
-### HTTP Archive
+<dt id="goja">Goja</dt>
+<dd>A JavaScript runtime, written in pure Go, that emphasizes standard compliance and performance. k6 uses Goja to enable test scripting without compromising speed, efficiency or reliability, compromises that would have been necessary with NodeJS
+<p>Read more: [Goja repository on GitHub](https://github.com/dop251/goja).</p></dd>
 
-An **HTTP Archive**, or **HAR file**, is a file containing logs of a browser interactions with the system under test. All of the included transactions are stored as JSON-formatted text. These archives may then be used to generate test scripts using, for instance, the [har-to-k6 Converter](https://github.com/k6io/har-to-k6). For more details, see the [HAR 1.2 Specification](http://www.softwareishard.com/blog/har-12-spec/).
+<dt id="graceful-stop">Graceful stop</dt>
+<dd>A period that lets VUs finish an iteration at the end of a load test. Graceful stops prevent abrupt halts in execution, and graceful ramp downs prevent unrealistic drops to zero VUs.
+<p>Read more: The [Graceful stop reference](https://k6.io/docs/using-k6/scenarios/graceful-stop/)</p></dd>
 
-### Iteration
+<dt id="horizontal-scalability">Horizontal scalability</dt>
+<dd>The degree to which one can improve the performance of a system by adding more nodes (servers or computers for instance).</dd>
 
-An iteration is an execution of the `default function`, or scenario `exec` function.
+<dt id="http-archive-(har-file)">HTTP archive (HAR file)</dt>
+<dd>A file containing logs of a browser interactions with the system under test. All included transactions are stored as JSON-formatted text. You can use these archives to generate test scripts using (for example, with the har-to-k6 Converter).
+<p>Read more: [HAR 1.2 Specification](http://www.softwareishard.com/blog/har-12-spec/), [HAR converter](https://k6.io/docs/test-authoring/recording-a-session/har-converter/)</p></dd>
 
+<dt id="iteration">Iteration</dt>
+<dd>A single run in the execution of the `default function`, or scenario `exec` function.
 You can either calculate iterations across all [virtual users](#virtual-users) (as done by the [Shared iterations](/using-k6/scenarios/executors/shared-iterations) executor), or per virtual user (as the [Per-VU Iterations](/using-k6/scenarios/executors/per-vu-iterations)).
```

Comment:
> You can either calculate iterations across all [virtual users](#virtual-users) (as done by the [Shared iterations](/using-k6/scenarios/executors/shared-iterations) executor), or per virtual user (as the [Per-VU Iterations](/using-k6/scenarios/executors/per-vu-iterations)).

I am not sure about so much details on the description.

Reply:
I trimmed this but I kept in the essential detail that they can be shared between VUs or per VU. Perhaps a user who's confused about that may consult the glossary to see what an iteration really is.

Reply:
Good point; it was ambiguous.

> You can set iterations across all VUs, or per VU.

But if you think that's unneeded detail, I'm fine with just trimming that sentence off.